src/scripts/main.js

```diff
@@ -1 +1,56 @@
 'use strict';
+
+function notification(className, text) {
+  document.body.insertAdjacentHTML('afterbegin', `
+  <div class="${className}" data-qa="notification">
+  ${text}
+</div>
+  `);
+}
+
+const promise1 = new Promise((resolve, reject) => {
+  document.addEventListener('mousedown', () => {
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(() => {
+    reject(new Error('First promise was rejected'));
+  }, 3000);
+});
+
+promise1
+  .then(result => notification(result, 'success'))
+  .catch(error => notification(error, 'warning'));
```

Comment:
notification text shouldn't be just 'success' or 'warning'
you have the text in result and  error variable

Reply:
done